spec/rubocop/cop/rspec/expect_in_let_spec.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::RSpec::ExpectInLet do
+  it 'adds an offense for `expect` in let' do
+    expect_offense(<<~RUBY)
+      let(:foo) do
+        expect(something).to eq 'foo'
+        ^^^^^^ Do not use `expect` in let
+        is_expected.to eq 'foo'
+        ^^^^^^^^^^^ Do not use `is_expected` in let
+        expect_any_instance_of(Something).to receive :foo
+        ^^^^^^^^^^^^^^^^^^^^^^ Do not use `expect_any_instance_of` in let
+      end
+    RUBY
+  end
+
+  it 'accepts an empty let' do
+    expect_no_offenses(<<~RUBY)
+      let(:foo) {}
+    RUBY
+  end
```

Comment:
Can you please add an example where ‘expect’ is the only statement in the ‘let’ block?

Reply:
Probably just split the example in line 4 into three separate examples?

Reply:
Thank you for reviewing!
Should I keep the original source example that I split?

Reply:
The point is that the AST and the node pattern for single-statement blocks and multi-statement blocks are different.
I’m being extra cautious here and have examples (where offences are added) for both cases.

Another case is when there are multiple offences (for multiple statements) all offensive statements are marked as such.

Basing on this, I suggest to structure this as:
- several examples with one statement each (you have those)
- One example with a combination of two statements (please add)
- One example with no offences and an empty ‘let’ block (you have this)
- One example with ‘it’ and no offences (you have this)

I’m happy to explain it further. Please correct me if I’m talking nonsense - it’s an evening of a dense day

And thanks for the persistence!

Reply:
> The point is that the AST and the node pattern for single-statement blocks and multi-statement blocks are different.

I see.  Thank you.

> One example with a combination of two statements (please add)

I did that.